src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -54,13 +54,15 @@
                 box-shadow: $bdl-header-box-shadow;
             }
 
-            .submenu {
+            .submenu:not(.is-hidden) {
```

Comment:
what happens if we remove `:not(.is-hidden)`?

I'm wondering if the extra specificity here will take precedence: https://github.com/box/box-ui-elements/blob/00fc868b7f67566927eaaa11a6c5f7eb24438961/src/components/menu/SubmenuItem.scss#L20-L23

Reply:
It doesn't take precedence

Reply:
These styles only really apply when the submenu is not hidden anyways and it was easier to just omit is-hidden to not have to deal with style overrides